### ISSUE TITLE
Using spqrel_navigation with virtual robot

### ISSUE THREAD

**Issue by bornaZbodulja** [LCAS/spqrel_navigation#12]
Hi,

I'm doing some research regarding navigation solutions with pepper robot. Because of the current situation I'm not able to work on a real robot so I was wandering is there any way that I could test spqrel in some simulator like gazebo or qibullet? 

Thanks

**Comment by MFernandezCarmona** [comment]
Hi,
I'm afraid never tried this, but it is a really interesting topic and I encourage you to try. I'd say you go for qibullet, as it looks to me the one that will give you the closest interface to the real one.
Sorry if we can't help you more.
Best

**Comment by bornaZbodulja** [comment]
Ok, thanks I'll look into it. Just one question: can you use the same gazebo environment in qibullet because I already have map of that  environment?

**Comment by MFernandezCarmona** [comment]
Not exactly. Qibullet supports loading URDF files, but gazebo uses "world" files. You will have to dig a bit on how to transfer your gazebo map into an URDF.
